README.md

```diff
@@ -128,6 +130,11 @@ The behavior of some of the views had to be modified to address functionalities
     # -t Name and optionally a tag in the <image-name>:<image-tag> format
     ```
 
+    ```bash
+    # e.g.
+    docker build -t trucksign:0.0.0.1 .
```

Comment:
why the 4th digit? semantic versions are usually split into MAJOR.MINOR.PATCH versions which is why its 3 digits in 1.0.0

just randomly asking

Reply:
I didn't even know that there was a standard xDDD